src/lib/factory.js

```diff
@@ -8,6 +8,7 @@ const request = require('co-request')
 const pathToRegexp = require('path-to-regexp')
 const EventEmitter2 = require('eventemitter2').EventEmitter2
 const translateBellsToPluginApi = require('./translate').translateBellsToPluginApi
+const Url = require('url')
```

Comment:
Pretty odd to use upper-case here. Node modules that are static objects (not classes) are usually assigned to mixedCase variables: https://nodejs.org/api/url.html#url_url